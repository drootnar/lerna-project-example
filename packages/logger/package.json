{
  "name": "@lerna-pro/logger",
  "version": "0.0.1",
  "description": "lerna-pro logger",
  "license": "ISC",
  "main": "lib/index.js",
  "scripts": {
    "test": "jest --logHeapUsage",
    "test-watch": "jest --watch --logHeapUsage",
    "lint": "eslint src/**/*.js test/**/*.js"
  },
  "devDependencies": {
  },
  "dependencies": {
    "@lerna-pro/config": "^0.0.1"
  }
}
