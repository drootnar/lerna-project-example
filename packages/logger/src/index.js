import config from '@lerna-pro/config'

export default config
