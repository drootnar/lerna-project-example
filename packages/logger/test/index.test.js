import logger from '../src/index'

describe('logger', () => {
  it('temp test', async () => {
    expect(logger).toMatchSnapshot()
  })
})
