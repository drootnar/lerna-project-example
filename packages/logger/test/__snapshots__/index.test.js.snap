// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logger temp test 1`] = `
Object {
  "logLevel": "debug",
}
`;
